Atomic/atomic.py

```diff
@@ -648,7 +648,7 @@ def _no_such_image():
         inspection = None
         if not self.args.force_remote_info:
             try:
-                inspection = self.d.inspect_image(self.args.image)
+                inspection = self._inspect_image(self.args.image)
```

Comment:
Ahh.. So the original error message was good enough, but there was a bug right here where we weren't actually calling our internal version of `_inspect_image()`. Cool.